### PR TITLE
Updating Readme with initial content and scope

### PR DESCRIPTION
**Pull request by robscott**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
+This project is early in the development phase and is still experimental in
+nature. Both bugs and breaking changes are likely.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+## Scope
 
-You can reach the maintainers of this project at:
+This project is primarily focused on translating Ingress resources to Gateway
+API resources. Some widely used annotations and/or CRDs _may_ be supported, as
+long as they can be translated to Gateway API directly. This project is not
```
Comment (sunjayBhatia) [inline README.md:16]: that sounds great to me, from Contour's perspective we explicitly stopped adding annotations to support features in Ingress and prefer first-class fields in HTTPProxy, as well as first-class fields in HTTPRoute (and future extension CRDs)

**Review by sunjayBhatia — APPROVED** [review]

**Comment by robscott** [comment]
Thanks for the feedback @shaneutt and @sunjayBhatia! Can one of you LGTM this one? I'll send a follow up with some actual code next.

**Review by shaneutt — APPROVED** [review]
/lgtm

**Pull request merged**